src/eml/main.ml

```diff
@@ -21,10 +21,16 @@ struct
   let workspace_path =
     ref ""
 
+  let emit_reason =
+    ref false
+
   let options = Arg.align [
     "--workspace",
     Arg.Set_string workspace_path,
     "PATH Relative path to the Dune workspace for better locations";
+    "--emit-reason",
+    Arg.Set emit_reason,
+    "Emit Reason syntax after preprocessing the template";
```

Comment:
I recommend adding a space at the beginning of the string, so

```
" Emit Reason syntax after preprocessing the template"
```

`Arg.align` (called on this list) will [use that space to align the string](https://ocaml.org/api/Arg.html#VALalign). At the moment,

```
$ dune exec src/eml/main.exe -- --help

Usage:

  eml FILE

  --workspace PATH   Relative path to the Dune workspace for better locations
  --emit-reason Emit Reason syntax after preprocessing the template
  -help              Display this list of options
  --help             Display this list of options
```

it should be

```
dune exec src/eml/main.exe -- --help

Usage:

  eml FILE

  --workspace PATH   Relative path to the Dune workspace for better locations
  --emit-reason      Emit Reason syntax after preprocessing the template
  -help              Display this list of options
  --help             Display this list of options
```

and the extra space will fix this.